site/docs/v1/tech/identity-providers/nintendo.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: doc
+title: Nintendo Identity Provider
+description: Login with Nintendo using the Nintendo Identity Provider
+---
+
+:sectnumlevels: 0
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.36.0.
+====
+
+== Overview
+
+This section will cover how to add a Login with Nintendo button to FusionAuth. Below is an example login page with the Nintendo Identity Provider enabled.
+
+* <<Create a Nintendo Developer Account>>
+* <<Create a Nintendo Identity Provider>>
+* <<Building Your Own Integration>>
+
+{empty} +
+
+Once you have completed this configuration you will be able to enable the Nintendo login button for one or more FusionAuth Applications. Below is an example login page with the Nintendo Identity Provider enabled.
+
+image::identity-providers/login-nintendo.png[Nintendo Login]
```

Comment:
I think the rest of the IdP examples have a cropped image here.

Reply:
I did actually crop it, but accidentally saved the cropped version in _site instead of site. Fixing it now.

Reply:
Should be fixed now.